packages/unmock-core/src/__tests__/util.test.ts

```diff
@@ -89,6 +89,11 @@ test("build correct path without ignore", () => {
   expect(searchParams.get("headers")).toBe(
     '{"Authorization":"Foo","User-Agent":"Bar"}',
   );
+  expect(JSON.parse(searchParams.get("actions") || "")).toEqual([
```

Comment:
Is the `|| ""` needed for `toEqual`? Can't we use `toBe` instead if so, just to make the test clearer?